src/main/java/version1/gameUtil/screens/MazeScreen.java

```diff
@@ -97,10 +101,16 @@ public void ready() {
      * @return this
      */
     public MazeScreen createGoToMenuButton(String label){
```

Comment:
you can delete this method, you only need one mode button. Two have been created